after.sh

```diff
@@ -20,12 +23,23 @@ psql -hlocalhost -Uhomestead -dms_local -w -c 'create schema if not exists data_
 
 # SETUP ms-emails
 psql -hlocalhost -Uhomestead -dms_local -w -c 'create schema if not exists emails;'
+psql -hlocalhost -Uhomestead ms_local < /home/vagrant/laravel/ms-emails/storage/emails.sql
 
 # SETUP ms-five9
 psql -hlocalhost -Uhomestead -dms_local -w -c 'create schema if not exists five9;'
 
+# SETUP ms-geo
+psql -hlocalhost -Uhomestead -dms_local -w -c 'create schema if not exists geo;'
```

Comment:
@stephen-ta I modified my `after.sh` locally so I could test ms-geo locally, and I ended up with the following for the ms-geo setup:
```
# SETUP ms-geo
# install postgis
sudo apt-get -y -q install postgis
psql -hlocalhost -Uhomestead -dms_local -w -c 'create schema if not exists geo;'
PGPASSWORD=secret psql -hlocalhost -Uhomestead -dms_local -w -c 'create extension postgis schema public;'
```
I don't know if you want/need to add this

Reply:
@ssasarita That's great! Thank you!